### DISCUSSION TITLE
Browsing

### DISCUSSION THREAD

**Discussion by noahfraiture** [muety/wakapi#579] [Q&A]
Hey, I have a simple problem. When I put my url of self-host (`http://localhost:3000/api`), the same that I have in the wakatime config file on my system, in my browser, it tell me this is a wrong api url or api key. Do you know if it's normal and how I can fix that ? Maybe http isn't allowed from the browser and I have to put a reverse proxy ? I host the wakapi server on my own computer so I didn't find utility to put a reverse proxy

**Comment by muety** [comment]
Hi @NoahFraiture, `localhost` and HTTP (without TLS) shouldn't be a problem. But you'll have to put `http://localhost:3000/api/compat/wakatime/v1` as the API URL.

**Reply by noahfraiture** [reply]
Oh thank you ! That's good help
